mjcf_to_sdformat/mjcf_to_sdformat/converters/world.py

```diff
@@ -75,6 +77,18 @@ def iterate_bodies(input_body,
                                     physics,
                                     body_parent_name=body_parent_name,
                                     modifiers=modifiers)
+            for joint in body.joint:
```

Comment:
If a body has multiple joints, Mujoco puts them in series with eachother. 
However, the code here puts them in parallel. Since SDFormat doesn't support 
joints connected in series, we need to add dummpy links in between the joints. 
For example, if we have the following MJCF:

```xml
<body name="body1">
  <body name="body2">
    <joint name="joint1"/>
    <joint name="joint2"/>
  </body>
</body>
```

The resulting SDFormat should look something like this:

```xml
<model name="...">
  <link name="body1">
    ...
  </link>
  <joint name="joint1" type="prismatic">
    <parent>body1</parent>
    <child>body2_dummy1</child>
    ...
  </joint>
  <link name="body2_dummy1">
    <inertial>
      <mass>1e-6</mass>
      <inertia>
        <ixx>1e-6</ixx>
        <iyy>1e-6</iyy>
        <izz>1e-6</izz>
      </inertia>
    </inertial>
  </link>
  <joint name="joint2" type="prismatic">
    <parent>body2_dummy1</parent>
    <child>body2</child>
    ...
  </joint>
  <link name="body2">
    ...
  </link>
```
We might have to adjust the mass and inertia values, but these should work for 
now.

I suggest we also add a warning that dummy links with small masses are being 
added when we encounter joints in series.

Reply:
I will open a new PR for this